src/__tests__/flow.node.js

```diff
@@ -0,0 +1,24 @@
+/** Copyright (c) 2018 Uber Technologies, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import execa from 'execa';
+import test from 'tape-cup';
+
+test('Flow tests', async t => {
+  const failurePath = 'src/fixtures/failure/';
+  const successPath = 'src/fixtures/success/';
+  try {
+    await execa.shell(`yarn flow check ${failurePath}`);
```

Comment:
Why do we need to test this? Isn't it basically testing that Flow works? Or is there something else to it I'm missing?

Reply:
Ah makes sense